### PR TITLE
refactor: Tweaks variable names to reduce code.

### PR DESCRIPTION
**Pull request by usualoma**
It would reduce the number of conditional branches in `result?.handlers` and `result?.params` and shorten the code by a few lines.

Since this is not a particularly big improvement, it is safe to decide not to merge.

### CODE REVIEW

**Comment by yusukebe** [comment]
Hi @usualoma!

It's neat and it's great that `result?` has been removed! I'd like to merge this, but there's a conflict. Could you resolve it?

File: src/context.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class Context<
   private _pH: Record<string, string> | undefined = undefined // _preparedHeaders
   private _res: Response | undefined
   private _path: string = '/'
-  private _pData?: Record<string, string> | null // __paramData
+  private _params?: Record<string, string> | null // __params
```
Comment (yusukebe) [inline src/context.ts:92]: The comment `// __params` is not necessary.

Comment (usualoma) [reply src/context.ts]: Fixed!
0e39109

**Comment by usualoma** [comment]
@yusukebe Thanks! Resolved.  6c0b358

**Comment by yusukebe** [comment]
Thanks! Merge now!

**Pull request merged**